src/views/layouts/main.hbs

```diff
@@ -1,15 +1,66 @@
-<html lang="en">
+<html lang="en" dir="ltr">
   <head>
     <link
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
     />
-    <title></title>
+    <title>
+      {{#if title}}
+        {{title}} - Frontend Forward 2024
+      {{^}}
+        Frontend Forward 2024
+      {{/if}}
+    </title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="view-transition" content="same-origin" />
```

Comment:
This allows for cross-document view transitions. Without it it would only work for SPAs